docs/android/v2/guides/token-endpoint.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Auth Token Endpoint Guide
+nav: 2.3
+---
+
+## Get an app token using token endpoint
+
+You can use the token endpoint from your 100ms dashboard while building integration with 100ms. This acts as a tool enabling front-end developers to complete the integration without depending on the backend developers to set up a token generation backend service.
+
+**URL format:** `https://prod-in2.100ms.live/hmsapi/<your-100ms-subdomain>.app.100ms.live/api/token`
+
+100ms token endpoint can generate an app token with the inputs passed, such as room_id, role, & user_id (optional - your internal user identifier as the peer's user_id). You can use [jwt.io](https://jwt.io/) to validate whether the app token contains the same input values.
+
+<PostRequest title="https://prod-in2.100ms.live/hmsapi/johndoe.app.100ms.live/api/token" />
+<Request id="req-comp-0">
+
+```bash
+curl --location --request POST 'https://prod-in2.100ms.live/hmsapi/johndoe.app.100ms.live/api/token' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "room_id":"633fcdd84208780bf665346a",
+    "role":"host",
+    "user_id":"1234"
+}'
+```
+
+</Request>
+<ResponseBox id="resp-0" status="200 OK">
+
+```json
+{
+    "token": "***************************",
+    "msg": "token generated successfully",
+    "status": 200,
+    "success": true,
+    "api_version": "2.0.192"
+}
+```
+
+</ResponseBox>
+
+### Example client-side implementation
+
+You can directly add this to your client-side implementation, for example:
+
+```jsx
+const response = await fetch(`${tokenEndpoint}api/token`, {
```

Comment:
more useful to link to the sample app code where this is called because that code snippet is in platform language